burner-ui/src/components/HistoryListEvent.tsx

```diff
@@ -60,11 +58,8 @@ const HistoryListEvent: React.FC<HistoryListEventProps> = ({
           </div>
 
           <div>
-            <Text level={2} as={'div'} right color={event.to === account ? '#28C081' : '#FD9D28'}>
```

Comment:
Your branch here is deleting the conditional color styles I added into `develop`. What's the best way to keep this in while respecting the changes you made to your branch?

Reply:
I'll rebase develop